scripts/travis-screenshots

```diff
@@ -0,0 +1,24 @@
+#! /bin/bash
+
+# Decrypt the deployment key
+openssl aes-256-cbc -K $encrypted_1645300b04d0_key -iv $encrypted_1645300b04d0_iv -in .travis.sskey.enc -out .travis.sskey -d
+
+# Configure ssh access using the key
+mv .travis.sskey ~/.ssh/id_rsa_deploy
```

Comment:
The token system allows you to post to the same repository that you are building (in this case `skytruth/pelagos-client`), but does not allow you to post to other repos. You need deployment keys for that. The alternative was using a personal token, and encrypting that token using `travis encrypt`, but as I already had to change it, I preferred using the recommended github deployment keys instead.